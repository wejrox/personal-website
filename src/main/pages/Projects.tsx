import * as React from "react";

class Projects extends React.Component {
    public render() {
        return (<div>Projects</div>);
    }
}

export default Projects;
