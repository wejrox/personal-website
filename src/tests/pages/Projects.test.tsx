import React from "react";
import * as Enzyme from "enzyme";
import Projects from "../../main/pages/Projects";

describe("Projects Page", () => {
    const wrapper = Enzyme.shallow(<Projects/>);

    it("renders without crashing", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
