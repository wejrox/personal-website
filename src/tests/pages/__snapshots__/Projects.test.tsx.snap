// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects Page renders without crashing 1`] = `ShallowWrapper {}`;
